src/project.ts

```diff
@@ -20,8 +44,16 @@ export interface GitHubActionTypeScriptOptions extends typescript.TypeScriptProj
  * @pjid github-action-ts
  */
 export class GitHubActionTypeScriptProject extends typescript.TypeScriptProject {
+  public actionMetadata: GitHubActionMetadata;
+  public actionName: string;
```

Comment:
I'd prefer making this `readonly`, or making a private value with getters/setters otherwise.